website/docs/registry/faq.mdx

```diff
@@ -120,3 +120,9 @@ If your GPG key or Github account has been compromised, please [reach out to us
 ## Can I regain ownership of my artifact after losing access to my GitHub account?
 
 For support, please [reach out to us via email](/terraform/registry#getting-help).
+
+## Documentation Markdown Support
```

Comment:
If you don't like my above suggestion, then I'd recommend removing this header in favor of converting the h3 below into an h2. Then the question would match the rest of this page's format:

```suggestion
```

(for visual reference here is the right-hand TOC )
![Screenshot 2023-12-20 at 11 00 11 AM](https://github.com/hashicorp/terraform-docs-common/assets/32436232/c175c136-6429-45dc-a48d-ed8825461b69)